src/components/Board.js

```diff
@@ -202,34 +207,34 @@ class Board extends Component {
     /**
      * Pause or Resume Game Timer
      */
-    pauseOrPlay() {
-        let { paused } = this.state;
-
-        if (paused === true) {
-            this.setState({paused: false});
-            this.incrementTimer();
-        } else {
-            this.setState({paused: true});
-            clearInterval(this.timer);
-        }
-        this.boardPlay(paused);
-    }
+    // pauseOrPlay() {
```

Comment:
Should remove this if you're commenting this out

Reply:
Ok, I fixed it.